Makefile

```diff
@@ -1,2 +1,41 @@
-test: 
-	TF_ACC=1 go test ./logdna -v 
+TEST?=$$(go list ./... | grep -v 'vendor')
+COVERAGEFILE?=coverage.out
+HOSTNAME=logdna.com
+NAMESPACE=logdna
+NAME=logdna
+BINARY=terraform-provider-${NAME}
+VERSION=1.1.0
+OS_ARCH=darwin_amd64
+
+default: install
+
+build:
+	go build -o ${BINARY}
+
+release:
```

Comment:
this looks good, but any thoughts on using `goreleaser` with their [recommended config](https://github.com/hashicorp/terraform-provider-scaffolding/blob/main/.goreleaser.yml)? happy to punt on this topic until we're ready though.